genestack_client/scripts/genestack_user_setup.py

```diff
@@ -260,10 +300,11 @@ class Init(Command):
     def get_command_parser(self, parser=None):
         parser = parser or ArgumentParser(description=self.DESCRIPTION)
         parser.description = self.DESCRIPTION
-        group = parser.add_argument_group("command arguments")
+        group = parser.add_argument_group('command arguments')
         self.update_parser(group)
         group.add_argument('-H', '--host', default=DEFAULT_HOST,
-                           help="server host, use it to make init with different host, default: %s" % DEFAULT_HOST,
+                           help='server host, use it to make init with different host,'
+                                ' default: %s' % DEFAULT_HOST,
```

Comment:
That text really baffles me, what does it mean?
"Genestack host, change it to connect somewhere else than $DEFAULT"?